packages/terra-table/docs/guides/MultiSelectTable.md

```diff
@@ -0,0 +1,127 @@
+# Terra Table - Implementing a Multi Select Table
+
+In previous versions of the terra-table a multi select style table could be created as either a controlled version of SelectableTableRows or uncontrolled version of MultiSelectableRows.  These implementations suffered from inflexibility and performance concerns. Going foward terra-table is more granular, though this puts more reponsibility on the consumer to properly update their table rows with the appropriate state. The following is a guide to addressing those concerns in your implementation.
```

Comment:
I think the blur makes sense for an upgrade guide, but the background context might be confusing or too much for developer unfamiliar with the previous table version

Reply:
Updated to be more focused on implementation.